crates/sui/src/validator_commands.rs

```diff
@@ -454,6 +474,70 @@ impl SuiValidatorCommand {
                     serialized_data,
                 }
             }
+            SuiValidatorCommand::BridgeCommitteeRegistration {
+                bridge_node_config_path,
+                bridge_authority_url,
+                gas_budget,
+            } => {
+                let bridge_config = match BridgeNodeConfig::load(bridge_node_config_path) {
+                    Ok(config) => config,
+                    Err(e) => panic!("Couldn't load BridgeNodeConfig, caused by: {e}"),
+                };
+                // Read bridge keypair
+                let key_str =
+                    fs::read_to_string(bridge_config.bridge_authority_key_path_base64_raw)?;
+                let bytes = Base64::decode(key_str.trim())
+                    .expect("Decode authority key from file should not fail.");
+                let key = Secp256k1PrivateKey::from_bytes(&bytes)
+                    .expect("Decode authority key from bytes should not fail.");
+                let ecdsa_keypair = Secp256k1KeyPair::from(key);
```

Comment:
use https://github.com/MystenLabs/sui/blob/5db2bb5d86dd4e8e48678bfe08272fc5d37379a6/crates/sui-bridge/src/config.rs#L392

Reply:
@patrickkuo  one more comment here in case you missed it